client/src/bundles/vendor.js

```diff
@@ -11,7 +11,7 @@ require('json-js');
 
 // jQuery plugins require that the jQuery object is exposed as a global
 // webpack.ProvidePlugin is used to ensure that jQuery and $ are provided to all includes
-require('script-loader!../../../thirdparty/jquery/jquery.js');
+require('script-loader!../../../thirdparty/jquery/jquerys');
```

Comment:
I think there's a typo in the statement.

```suggestion
require('script-loader!../../../thirdparty/jquery/jquery');
```

Reply:
Ha fixed